cmd/server/flags.go

```diff
@@ -270,6 +270,12 @@ var flags = []cli.Flag{
 		Name:    "github-skip-verify",
 		Usage:   "github skip ssl verification",
 	},
+	&cli.StringFlag{
```

Comment:
not sure - should it be global or repo specific
---
cc @anbraten to our talk about more events yesterday ... -> something for  Monday (#198) ?

Reply:
Ah, well. Since each of the repos my container different actions. I'll keep it in the repo. That is where I would expect it to be when configuring.

Reply:
@LamaAni before adding this ... just leave it out? for this pull - and we should talk about it on  7. ... I'd like to make a dessission after we had some more thoughts in this - do you have time to join the meeting ?

Reply:
Sure. I'm on the discord. I have time in the next hour.

Reply:
Could we send both as `release` event and add an env-var like `CI_RELEASE_TYPE=prerelease` to the pipeline?

Reply:
I would prefer the CI_ACTION .. since for github all events have actions.

Reply:
For now this would allow the release-plugin / script to check on it and for the long term you could use sth like #312 to test if the step should be executed or not.

Reply:
@LamaAni -> discord